scripts/submitHistoFiller.py

```diff
@@ -3,62 +3,71 @@
 import argparse
 import datetime
 
+hourdate = datetime.datetime.now().strftime('%Y.%m.%d_%H.%M.%S').replace('.','-')
+
 parser = argparse.ArgumentParser(description='Command line parser of plotting options')
 parser.add_argument('--cfg', dest='cfg', required=True,
                     help='name of the main cfg to run', default=None)
-parser.add_argument('--tag', dest='tag', required=True, default=None,
+parser.add_argument('-o', '--outdir', dest='outdir', required=False,
```

Comment:
This makes sense. Just to point out the obvious: this would also affect the pipeline for histogram generation for 2016/17. 
@dzuolo @kramerto FYI, let us know if you're fine with the update

Reply:
Good point. The adaptation can follow what is done in ```config/mainCfg_*_UL18.cfg```, namely simply removing the line with ```outputFolder = ... ```.

Reply:
I'll have a look in the next days and check if I understand all the changes that are made and their implications. Thanks!

Reply:
@dzuolo Note that the LLR documentation is up-to-date; for the histogram production, for instance, see [this](https://github.com/LLRCMS/KLUBAnalysis/wiki/LLR-Workflow#histogram-production).